clients/spark/build.sbt

```diff
@@ -2,7 +2,7 @@ import build.BuildType
 
 lazy val baseName = "lakefs-spark"
 
-lazy val projectVersion = "0.1.0-SNAPSHOT"
+lazy val projectVersion = "0.1.4-SNAPSHOT"
```

Comment:
Is there a way to avoid overrides? It's such a common error to publish without advancing this value..